apps/frontend/src/components/Input.tsx

```diff
@@ -0,0 +1,38 @@
+import type { InputHTMLAttributes } from 'react'
+import { cn } from '../utils/classname'
+
+export const Input = ({
+  label,
+  error,
+  ...props
+}: {
+  label: string
+  error: string | null
+} & InputHTMLAttributes<HTMLInputElement>) => {
+  return (
+    <div className="flex flex-col gap-1">
+      <div
+        className={cn(
+          'relative flex w-full items-center gap-2 rounded-lg border border-transparent bg-slate-600 px-4 py-2 text-slate-50 outline-none has-focus-visible:border-blue',
+          error && 'border-red',
+        )}
+      >
+        <input
+          {...props}
```

Comment:
```suggestion
          {...props}
          placeholder={label}
```

i suggest we add the label as a placeholder since the component needs a placeholder to work properly but it will never be visible on the screen

Reply:
good idea!